addons/account/views/account_move_views.xml

```diff
@@ -632,8 +648,8 @@
                                 <field name="invoice_line_ids"
                                        widget="section_and_note_one2many"
                                        mode="tree,kanban"
-                                       context="{'journal_id': journal_id, 'default_partner_id': commercial_partner_id, 'default_currency_id': currency_id != company_currency_id and currency_id or False}">
-                                    <tree editable="bottom" string="Journal Items">
+                                       context="{'invoice_line_ids': invoice_line_ids, 'journal_id': journal_id, 'default_partner_id': commercial_partner_id, 'default_currency_id': currency_id != company_currency_id and currency_id or False}">
```

Comment:
Why adding `invoice_line_ids` in context? It don't seem to be used anywhere

Reply:
Indeed, I forgot to remove it after a change of specs